Tools/TestWebKitAPI/Tests/mac/ContextMenuTests.mm

```diff
@@ -831,6 +831,15 @@ static void swizzledOrderFrontColorPanel(id, SEL, id)
     Util::run(&gotProposedMenu);
 }
 
+TEST(ContextMenuTests, HitTestResultLinkWithInvalidURL)
+{
+    RetainPtr webView = adoptNS([[TestWKWebView alloc] initWithFrame:NSMakeRect(0, 0, 400, 400)]);
+    [webView synchronouslyLoadHTMLString:@"<a href='https://' style='font-size: 100px;'>Link</a>"];
+
+    RetainPtr<_WKContextMenuElementInfo> elementInfo = [webView rightClickAtPointAndWaitForContextMenu:NSMakePoint(50, 350)];
```

Comment:
```suggestion
    RetainPtr elementInfo = [webView rightClickAtPointAndWaitForContextMenu:NSMakePoint(50, 350)];
```

Reply:
I deliberately did it since the return type wasn't obvious 😛

Reply:
okay!